outputs/mixins.py

```diff
@@ -635,7 +636,11 @@ def write_data(self, worksheet):
         self.write_header(worksheet, fields, iterative_sets_fields)
 
         # write content
-        self.write_content(worksheet, fields, iterative_sets_fields, objects)
+        paginator = Paginator(objects, 1000)
```

Comment:
Add a new method, for example `get_paginator()` returning either a Paginator or None value. If None value is returned, do not use paginating.